doc/user-stories/expert-partitioner.md

```diff
@@ -417,3 +420,143 @@ The specs are based on current version of expert partitioner for TW. The goal is
 * allows to remove one subvolume from the list
 * alerts when trying to create a subvolume that does not start by /@
   * automatically appends /@ to the subvolume path
+
+### When 'RAID' is selected in the tree view
+* shows a table with the columns: Device, Size, F, Enc, Type, FS Type, Label, Mount Point, RAID Type, Chunk Size
+  * and the table is filled out with info of all RAID devices
+* *(pending) shows the actions: 'Add RAID', 'Edit', 'Resize' and 'Delete'*
+* *(pending) and action 'Add RAID' is selected*
+  * and there is less than two not mounted partitions
+    * shows an error message
+  * and there is at least two not mounted partitions with correct partition ID (not windows)
+    * shows a wizard to create a new RAID
+* *(pending) and action 'Edit' is selected*
+  * shows dialog to edit the RAID partition
+* *(pending) and action 'Resize' is selected*
+  * shows a dialog to modify the RAID devices
+* *(pending) and action 'Delete' is selected*
+  * shows a confirm popup to delete the RAID
+  * deletes the RAID
+
+### *(pending) When a 'RAID' is selected in the tree view*
+* shows a view with two tabs: 'Overview' and 'Used Devices'
+* and tab 'Overview' is selected
+  * shows a report with three sections: 'Device', 'RAID' and 'File System'
+    * where 'Device' section contains the folling info
+      * device name
+      * device size
+      * encrypted
+      * device id
+    * where 'RAID' section contains the following info
+      * RAID type
+      * chunk size
+      * parity algorithm
+    * where 'File System' section contains the following info
+      * file system
+      * mount point
+      * label
+  * shows the actions: 'Edit', 'Resize' and 'Delete'
+    * and action 'Edit' is selected
+      * shows dialog to edit the RAID partition (similar to edit disk partition)
+    * and action 'Resize' is selected
+      * shows a dialog to modify the RADID devices
+    * and action 'Delete' is selected
+      * shows a confirm popup to delete the RAID
+      * deletes the RAID
+* and tab 'Used Devices' is selected
+  * shows a table with the columns: Device, Size, F, Enc and Type
+    * and the table is filled out with info of all devices that belong to the RAID
```

Comment:
Isn't worth mentioning than selecting one item in the table can be used to "jump" to the corresponding partition?

Reply:
Ohhh, I got just realized about that :)